src/GNDStk/Tree/test/meta.test.cpp

```diff
@@ -0,0 +1,180 @@
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+/*
+The file we'll be examining:
```

Comment:
Also, like before, it's there so that somebody who reads the queries can quickly see, without looking at the XML file in another window, how the queries align with the file content. Note also, as before, it's abbreviated ("...") versus the full file. I understand your sentiment, but also keep in mind that if we changed things so that we were reading a different file, then lots of other things - in particular all the queries - would have to change as well. I know, a failure to properly change the queries would result in errors, while comments are just comments. I guess I'm just not that worried that things are going to be changing in a way that would invalidate the comments without somebody noticing that it would do so (because they'd be in the .cpp file modifying other things anyway).

Reply:
I can of course remove comments like these if they really bother people a lot. I just happen to think they're useful, for the reason described, and are relatively unlikely to cause harm.